src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -517,6 +567,50 @@ export function GraphWrapper({
 							showing {graphRows.length} item{graphRows.length ? 's' : ''}
 						</span>
 					)}
+					<div className="actioncombo">
+						<button
+							type="button"
+							aria-controls="repo-columnsettings-list"
+							aria-expanded={columnSettingsExpanded}
+							aria-haspopup="listbox"
+							id="repo-actioncombo-label"
```

Comment:
using same `repo-` prefix for ids as the repo combobox

Reply:
Done in commit: `Fixed ids/keys prefix used for the columns combo`